core/src/main/java/com/linecorp/armeria/client/retry/Backoff.java

```diff
@@ -68,6 +68,14 @@ static Backoff exponential(long initialDelayMillis, long maxDelayMillis, double
         return new ExponentialBackoff(initialDelayMillis, maxDelayMillis, multiplier);
     }
 
+    /**
+     * Returns a {@link Backoff} for which the backoff delay increases in line with the Fibonacci sequence
+     * f(n) = f(n-1) + f(n-2).
```

Comment:
nit: how about adding `where f(1) = {@code initialDelayMillis}`

Reply:
I like it, added :) Thanks!